Makefile

```diff
@@ -118,6 +122,7 @@ else
 main.exe: $(OBJS) $(NXDK_DIR)/lib/xboxkrnl/libxboxkrnl.lib
 	@echo "[ LD       ] $@"
 	$(VE) $(LD) $(NXDK_LDFLAGS) $(LDFLAGS) -out:'$@' $^
+	$(VE) objcopy --rename-section 'XTIMAGE=$$$$XTIMAGE' --rename-section 'XSIMAGE=$$$$XSIMAGE' $@ || exit 0
```

Comment:
As thrimbor said, this probably doesn't need to be here because the makefile should never really build XBEs in the future.

Alternatively we could extend LLVM to allow such section names.

However, I think most people will just their own binutils line to inject a new section from a raw file, because to make *proper* use of these sections you'll want to use some tool which can encode the Xbox D3D image for these sections (so I doubt they exist as part of an EXE already, unless you have a specific case where you want to run CXBE on some XDK EXE.. but I'm not sure if anybody ever plans to do that or what those XDK EXE look like [if they have these sections for example]).

Reply:
I currently rely on calling NXDK's Makefile from my Makefile so I kind of need this
Also I made my icon by exporting the logo from GIMP as a .dds and overwriting the data from offset 0x800 in an existing .xpr image extracted from an xbe. I will probably make a tool to do this at some point.
I embed the icon by doing this in my main.c:
```
#if PLATFORM == PLAT_XBOX
__asm__ (
    ".section \"XTIMAGE\"\n"
    ".incbin \"icons/engine.xpr\""
);
#endif
```
The linker is a pain and will truncate section names to 8 chars and do some special stuff with `$`. Something like `my$ect` would turn into `my` and `verylongsectname` would turn into `verylong`.